protected/controllers/ModuleController.php

```diff
@@ -58,24 +58,23 @@ protected function performAjaxValidation($model)
 
     public function actionSaveLesson()
     {
-        $teacher = Yii::app()->user->getId();
-
-        $newOrder = Lecture::model()->addNewLesson(
-            $_POST['idModule'],
-            $_POST['titleUa'],
-            $_POST['titleRu'],
-            $_POST['titleEn'],
-            Teacher::model()->find('user_id=:user', array(':user' => $teacher))->teacher_id
+        $newLecture = array (
+            'titleUa' => Yii::app()->request->getParam('titleUa', ''),
```

Comment:
While we do it in the right way, could you please rename this variable like newLectureParams?